### ISSUE TITLE
Are you looking for a new maintainer?

### ISSUE THREAD

**Issue by ashb** [alex-rudakov/sphinx-argparse#135]
Hi there,

Love this project, and we use it on apache/airflow where it's great.

I notice that there are a number of open PRs, and I understand the burden of being an open-source maintainer, so I'd like to volunteer to help address some of these, get them merged/closed etc and then eventually released.

So I'm wondering if you are looking for help maintaining this project @alex-rudakov?

(I hope I wasn't to presumptuous in asking)

**Comment by dpryan79** [comment]
I think I'm the only (non-active on this project) maintainer left. I'm happy to add you since this isn't relevant enough to me any more that I can devote any significant amount of time to it.

**Comment by dpryan79** [comment]
Actually, it doesn't appear that I have admin privileges on this repos, only commit/tag/etc. privileges. On pypi I have maintainer access but can't manage the account. I'm happy to help however I can with transferring access on pypi. At least on github you can just fork the repo and be done with it.

**Comment by ashb** [comment]
I'm `ashb` on PyPi

**Comment by dpryan79** [comment]
@ashb Can you follow the procedure layed out in https://www.python.org/dev/peps/pep-0541/#how-to-request-a-name-transfer ? If you do that and post a link to it I can at least write in support of it. It took me a while to get in touch with the actual project owner a few years ago and I imagine it'd be even more difficult now. So that combined with the only listed maintainer supporting the request should (hopefully) facilitate things greatly.

**Comment by ashb** [comment]
@dpryan79 https://github.com/pypa/pypi-support/issues/905

**Comment by lucc** [comment]
What is your conclusion here: Will you continue development here or in a fork?

I was just hit by #115 but I think I found a fix and could open a PR.

**Comment by ashb** [comment]
I'm going to continue dev, yes, but I was hoping to not have to rename the module.

**Comment by JOJ0** [comment]
This is great news @ashb as I am just starting out with sphinx-argparse and was shocked about the status of maintenance status of this repo. It would be a shame if this stayed unmaintained. Good luck with taking over and don't feel yourself forced to rush it! Not having to rename will benefit everyone, even if now everything takes more time! All the best! Jojo

**Comment by ashdwilson** [comment]
👍

**Comment by laf0rge** [comment]
@ashb why don't you start working in a fork of this github repo, consolidating/merging various PRs that have accumulated here.

Whether or not pypi will eventually grant you permission to take over is unrelated to the above.

The most important part is to ahve one repo where the back-log of patches/PRs is being processed and where development can continue.

Thanks for stepping up!

**Comment by pradyunsg** [comment]
FWIW, it might make life easier for PyPI folks, if there's a replacement ready -- because then it's less of "I intend to revive the project" and more of "I have revived the project, and would like the PyPI name".

**Comment by ashb** [comment]
Yes, good idea. Starting over at https://github.com/ashb/sphinx-argparse

**Comment by gaborbernat** [comment]
In case anyone is looking for an alternative rendering, and that's well maintained, check out https://pypi.org/project/sphinx-argparse-cli/, which I created after enquiring about similar here https://github.com/alex-rudakov/sphinx-argparse/issues/134

**Comment by ashb** [comment]
@gaborbernat Happy to use/help with your fork rather than having another.

In 134 you said:

> implementation optimized for sub commands

Is that still the case? Does it work for normal commands just as well as this module does?

**Comment by gaborbernat** [comment]
Yes it does. 👍

**Comment by gaborbernat** [comment]
> Yes it does. 👍

It's more of a new implemention than fork 😁

**Comment by ashb** [comment]
@dpryan79 Are you able to grant me permissions on https://readthedocs.org/projects/sphinx-argparse/ please?
